ui/index.d.ts

```diff
@@ -459,7 +459,7 @@ export declare abstract class AzureParentTreeItem<TRoot extends ISubscriptionCon
 /**
 * Combines the root.environment.portalLink and id to open the resource in the portal.
```

Comment:
I try to keep these docs as up to date as possible. Maybe add a note like this:
> If `root` is a tree item, it will find the subscription ancestor and get environment.portalLink from there